frontend/app/src/theme/types.ts

```diff
@@ -15,6 +15,7 @@ export interface Theme {
     bg: string;
     txt: string;
     "txt-light": string;
+    "txt-muted": string;
```

Comment:
I think it's confusing and will cause future problems to introduce a  `txt-muted` field that is actually only used for something that is _not_ muted text. I had in mind something specific to what this is actually used for. i.e. something like `currentChat.msg.separator`.  That will definitely cause fewer problems if themes want to do different things.

I think it may also be impossible to get away with a single value. You might need a different value for your own messages and for other users messages. I don't think it's alway going to work to use the same colour in both cases.